app/forms/providers/proceeding_merits_task/attempts_to_settle_form.rb

```diff
@@ -0,0 +1,21 @@
+module Providers
+  module ProceedingMeritsTask
+    class AttemptsToSettleForm
+      include BaseForm
+
+      form_for ::ProceedingMeritsTask::AttemptsToSettle
+
+      attr_accessor :attempts_made, :application_proceeding_type_id
```

Comment:
Just wondering why you have :application_proceeding_type_id as an accessor ?

Reply:
Nice catch, as discussed this has been removed and is no longer needed 👍